putty.go

```diff
@@ -53,7 +58,99 @@ type Key struct {
 	Comment           string
 	Encryption        string
 	PrivateMac        []byte
-	decrypted         bool
+	padded            bool
+	keySize           int
+}
+
+func (k Key) Clone() *Key {
+	return &Key{
+		Version:           k.Version,
+		Algo:              k.Algo,
+		PublicKey:         k.PublicKey,
+		PrivateKey:        k.PrivateKey,
+		KeyDerivation:     k.KeyDerivation,
+		Argon2Memory:      k.Argon2Memory,
+		Argon2Passes:      k.Argon2Passes,
+		Argon2Parallelism: k.Argon2Parallelism,
+		Argon2Salt:        k.Argon2Salt,
+		Comment:           k.Comment,
+		Encryption:        k.Encryption,
+		PrivateMac:        k.PrivateMac,
+		//padded:            k.padded,
```

Comment:
why did you comment `padded` member?

Reply:
This is a mistake, should not be commented!